rotkehlchen/chain/evm/types.py

```diff
@@ -131,3 +133,26 @@ class RemoteDataQueryStatus(Enum):
     FAILED = auto()
     NEW_DATA = auto()
     NO_UPDATE = auto()
+
+
+class EvmIndexer(SerializableEnumNameMixin):
+    ETHERSCAN = auto()
+    BLOCKSCOUT = auto()
+    ROUTESCAN = auto()
+
+
+DEFAULT_EVM_INDEXER_ORDER: Final = (
+    EvmIndexer.ETHERSCAN,
+    EvmIndexer.BLOCKSCOUT,
+    EvmIndexer.ROUTESCAN,
+)
+CHAINS_WITH_PAID_ETHERSCAN: Final = {ChainID.BASE, ChainID.OPTIMISM, ChainID.BINANCE_SC}
```

Comment:
add a comment here linking to the etherscan announcement. Note this may be temporary and be changed by them so we would need to adjust here